utils/lxc/patches/020-lxc-checkconfig.patch

```diff
@@ -1,19 +1,26 @@
 --- a/src/lxc/cmd/lxc-checkconfig.in
 +++ b/src/lxc/cmd/lxc-checkconfig.in
-@@ -7,6 +7,16 @@ export LANGUAGE=en
+@@ -7,6 +7,15 @@ export LANGUAGE=en
  # Allow environment variables to override config
  : "${CONFIG:=/proc/config.gz}"
  : "${MODNAME:=configs}"
 +: ${ZGREP:=zgrep}
 +: ${GUNZIP:=gunzip}
 +
-+if [ -z $(command -v $ZGREP) ] && ! [ -z $(command -v $GUNZIP) ] && [ -x $(command -v $GUNZIP) ] &&  [ -f $CONFIG ] && [ "$CONFIG" == "/proc/config.gz" ] ; then
-+  CONFIG_NEW="/tmp/config-$(uname -r)"
-+  $GUNZIP -c $CONFIG > $CONFIG_NEW
-+  CONFIG=$CONFIG_NEW
-+
-+  GREP=grep
++if [ -z $(command -v $ZGREP) ] && ! [ -z $(command -v $GUNZIP) ] && [ -x $(command -v $GUNZIP) ] &&  [ -f $CONFIG ] && [ "$    CONFIG" == "/proc/config.gz" ] ; then
```

Comment:
could you please explain why do you make this (indentation) change?
and I believe `[ "$    CONFIG" == "/proc/config.gz" ]` is a new typo.

Reply:
You are right, fixed.